src/app/app.module.ts

```diff
@@ -12,6 +12,7 @@ import { MatIconModule } from '@angular/material/icon';
 import { MatMenuModule } from '@angular/material/menu';
 import { MatCardModule } from '@angular/material/card';
 import { FlexLayoutModule } from '@angular/flex-layout';
+import { RatingModule } from '../../node_modules/ngx-rating';
```

Comment:
You don't have to type out the relative path of the file, Angular will know to look at `/node_modules` if you don't specify a specific path.

Change it to `import { RatingModule } from 'ngx-rating';`